Submission ran but did not finish its execution due to the errors below. 
This does not mean the program did not execute however, it may or may not have produced some files or none: 

 Traceback (most recent call last):
  File "***Removed due to privacy***/decrypt_m50747jh/./decrypt_m50747jh.py", line 98, in <module>
    f.write(str(decrypt_morse(new_code1, " ")))
  File "***Removed due to privacy***/decrypt_m50747jh/./decrypt_m50747jh.py", line 83, in decrypt_morse
    morse_text = morse_text + morseList.get(text)
TypeError: can only concatenate str (not "NoneType") to str

Output for test_file 2 was incorrect
Output for test_file 3 was incorrect
